x/cash/controller_test.go

```diff
@@ -1,45 +1,43 @@
 package cash
 
 import (
-	"fmt"
 	"testing"
 
 	"github.com/iov-one/weave"
 	coin "github.com/iov-one/weave/coin"
 	"github.com/iov-one/weave/errors"
 	"github.com/iov-one/weave/store"
 	"github.com/iov-one/weave/weavetest"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
-func getWallet(kv weave.KVStore, addr weave.Address) coin.Coins {
+func wallet(t testing.TB, kv weave.KVStore, addr weave.Address) coin.Coins {
+	t.Helper()
+
 	bucket := NewBucket()
 	res, err := bucket.Get(kv, addr)
 	if err != nil {
-		panic(err) // testing only
+		t.Fatalf("cannot get wallet for %q: %s", addr, err)
```

Comment:
better.
even better would be `assert.Nil(t, err)`

Reply:
`assert.Nil` will not print the address, which might be helpful during the debugging.